tests/specs/api.js

```diff
@@ -146,6 +148,30 @@ test('store.reset creates empty instance of store with new options', function (t
   .catch(t.fail)
 })
 
+test('store.reset creates empty instance of store with new options passed as arguments', function (t) {
+  t.plan(1)
+
+  var CustomPouchDB = PouchDB.defaults({remote: 'test-db-clear-arguments-remote'})
```

Comment:
please use memory as backend. It’s a bit complicated, because different code is needed for browser / node. You can do this

```js
  var options = process.browser ? {
    adapter: 'memory'
  } : {
    db: require('memdown')
  }
  var CustomPouchDB = PouchDB.defaults(options)
```

The `remote: 'test-db-clear-arguments-remote'` option doesn’t do anything here